service-onboarding-build-pack/Jenkinsfile

```diff
@@ -390,20 +398,57 @@ node  {
 		}
 	}
 }
+/**
+ * For getting token to access catalog APIs.
+ * Must be a service account which has access to all services
+ */
+ def getAuthToken() {
+	 
+	withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: g_svc_admin_cred_ID, passwordVariable: 'PWD', usernameVariable: 'UNAME']]) {
+		echo "user name is $UNAME"
+
+		def loginUrl = g_base_url + "/platform/login"
+		def login_json = []
+
+		login_json = [
+			'username': UNAME,
+			'password': PWD
+		]
+
+		def payload = JsonOutput.toJson(login_json)
+
+		try {
+			def token = sh (script: "curl --silent -X POST -k -v \
+				-H \"Content-Type: application/json\" \
+				 $loginUrl \
+				-d \'${payload}\' | $jq -r '.data.token'", returnStdout:true)
+
+			token = token.replaceAll("[\n\r]", "")
+			echo "auth_token : $token"
```

Comment:
We should never print user tokens in the console. Please remove.

Reply:
Removed unwanted logs